src/components/Cell/Cell.tsx

```diff
@@ -37,14 +37,16 @@ const Cell = ({id, toggleModal}: GridCell) => {
         }
     }
 
+    const hoverStyle = isOver && !isDisabled && {transform: 'scale(1.3)'}
+
     const divStyle = {
         backgroundImage: `url(${cellContents?.plant.image})`,
         backgroundPosition: 'center',
         backgroundSize: '100%'
     };
 
     return (
-        <div id={id} className='cell' style={divStyle} onClick={handleClick} ref={dropRef}></div>
+        <div id={id} className='cell' style={Object.assign(divStyle, hoverStyle)} onClick={handleClick} ref={dropRef}></div>
```

Comment:
Rather than mutate the object itself via assign, you can splat them together like this:
```suggestion
        <div id={id} className='cell' style={{...divStyle, ...hoverStyle }} onClick={handleClick} ref={dropRef}></div>
```

Reply:
Random aside, but for other places where you're dynamically adding / removing classnames, this packages is what we use and may be useful for you all:
https://github.com/JedWatson/classnames

Reply:
awesomeeee I like that way better - will checkout that package!